frontend/src/components/pages/MultiUserBurndownChart/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, {useState} from "react";
+import Presentation from "./Presentation";
+import { connect } from "react-redux";
+import { fillInSnapshotDateGaps } from "./utils";
+
+export function MultiUserBurndownChartUnconnected({
+  //   mapStateToProps
+  currentTeamBurndownStats,
+  team,
+  metrics,
```

Comment:
So I was thinking of allowing us to use the metrics from the backend rather than hardcoding them in the frontend. The reason is if we want to add new metrics in the future, all we need to do is update the DB with new metrics. If we hardcode it on the frontend, then whenever we do want to add new metrics, we need to deploy a new version of the code.

@sheenarbw Is my thinking alright here or would we prefer to just hardcode the metrics on the frontend and leave it like that?